docs/filter_descriptions.md

```diff
@@ -2,33 +2,35 @@
 
 ### Video
 
-1.  [YUV Values/ MIN, AVG, MAX, LOW, HIGH](#yuv)
-2.  [YUV Values/ Difference](#diff)
-3.  [Saturation](#saturation)
-4.  [Hue](#hue)
-5.  [Temporal Outliers](#tout)
-6.  [Vertical Line Repetitions](#vrep)
-7.  [Broadcast Range](#brng)
-8.  [Crop Width (CropW)](#cropw)
-9.  [Crop Height (CropH)](#croph)
-10.  [Peak Signal to Noise Ratio (PSNRf)](#psnrf)
-11.  [Structural SImilarity Metric (SSIMf)](#ssimf)
-12.  [Mean Square Error (MSEf)](#msef)
-13.  [Interlacement Detection (single frame)](#singleframe)
-14.  [Interlacement Detection (multiple frames)](#multiframe)
-15.  [Interlacement Detection (repeated fields)](#repeated)
+[YUV Values/ MIN, AVG, MAX, LOW, HIGH](#yuv)  
+[YUV Values/ Difference](#diff)  
+[Saturation](#saturation)  
+[Hue](#hue)  
+[Temporal Outliers](#tout)  
+[Vertical Line Repetitions](#vrep)  
+[Broadcast Range](#brng)  
+[Crop Width (CropW)](#cropw)  
+[Crop Height (CropH)](#croph)  
+[Peak Signal to Noise Ratio (PSNRf)](#psnrf)  
+[Packet Duration Time (pkt_duration_time)](#pkd)  
+[Packet Size (pkt_size)](#ps)  
+[Structural SImilarity Metric (SSIMf)](#ssimf)  
+[Mean Square Error (MSEf)](#msef)  
+[Interlacement Detection (single frame)](#singleframe)  
+[Interlacement Detection (multiple frames)](#multiframe)  
+[Interlacement Detection (repeated fields)](#repeated)  
 
 ### Audio
 
-16.  [R.128](#r128)
-17.  [Audio Phase Meter](#phase)
-18.  [DC Offset](#dc)
-19.  [Audio Diffs](#audio-diffs)
-20.  [RMS](#rms)
+[R.128](#r128)  
```

Comment:
All of these should have trailing spaces according to Markdown rules of generating a new line.